src-unix/native/winsize.c

```diff
@@ -7,6 +7,11 @@
 #include <signal.h>
 #endif
 
+#ifdef __HAIKU__
+/* On some platforms, ioctl() is declared in <unistd.h>. */
+#include <unistd.h>
```

Comment:
Is including this unconditionally ok? @MisterDA @nojb what do you think?

Reply:
`unistd.h` does not exist on Windows, but should exist on Unix-like systems.

Reply:
AFAIK this is just a Haiku quirk and doesn't seem to be the case on other platforms.